tests/security/usbguard/usbguard.pm

```diff
@@ -0,0 +1,137 @@
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Summary: Test 'usbguard' can work:
+#          '# systemctl enable usbguard.service',
+#          '# systemctl start usbguard.service',
+#          '# systemctl status/is-active usbguard.service',
+#          '# usbguard generate-policy', '# usbguard allow-device',
+#          '# usbguard list-devices', '# usbguard block-device'
+#          '# check/cat '/var/log/usbguard/usbguard-audit.log',
+#          '# usbguard add-user', '# usbguard remove-user',
+#          '# usbguard get-parameter', '# usbguard set-parameter'
+#          '# usbguard generate/list/install/remove rules'
+# Maintainer: llzhao <llzhao@suse.com>
+# Tags: poo#102566, tc#1769830
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    my $out = '';
+    my $user = 'usbguard';
+    my $allow = 'allow';
+    my $block = 'block';
+    my $msg = 'Tablet';
+    my $f_rules_default = '/etc/usbguard/rules.conf';
+    my $f_rules_test = '/etc/usbguard/rules_test.conf';
+    my $f_IPCAccessControl = '/etc/usbguard/IPCAccessControl.d/\:' . "$user";
+    my $f_usbguard_audit_log = '/var/log/usbguard/usbguard-audit.log';
+
+    select_console 'root-console';
+
+    # 0. Set up environment
+    # Install usbguard packages
+    zypper_call('in libusbguard1 usbguard usbguard-devel usbguard-tools', timeout => 900);
+
+    # Start audit service
+    systemctl('restart auditd.service');
```

Comment:
Why use restart here?

Reply:
Just in case, "restart" is a better choice.

Reply:
I don't understand, why restart is better?